sockets/sockets.js

```diff
@@ -320,7 +322,57 @@ var actionsObj = {
 					return { message: (Math.floor(Math.random() * 10) + 1).toString(), classStr: "server-text" }
 				}
 			}
-		},
+        },
+        
+        mods: {
+            command: 'mods',
+            help: '/mods: Shows a list of online moderators.',
+            run() {
+                const modUsers = getPlayerUsernamesFromAllSockets().filter((username) => modsArray.includes(username));
+                const message = `Currently online mods: ${modUsers.length > 0 ? modUsers.join(', ') : 'None'}.`;
+                return { message, classStr: "server-text" };
+            }
+        },
+
+        pmmod: {
+            command: 'pmmod',
+            help: '/pmmod: Sends a private message to an online moderator.',
```

Comment:
Could you please fix this to:
             `help: '/pmmod <mod_username> <message>: Sends a private message to an online moderator.',`